src/scripting/ScriptContext.cpp

```diff
@@ -294,6 +304,38 @@ sol::object ScriptContext::GetRootObject() const
     return m_object;
 }
 
+sol::object ScriptContext::GetInputHandlers() const
+{
+    auto lockedState = m_sandbox.GetLockedState();
+    const auto& bindings = m_sandbox.GetBindings();
```

Comment:
I'd probably just use CET::Get().GetBindings() here, wouldnt necessarily make another GetBindings. If something needs them, they should be passed down (if it was done the right way). We have few places like this already, there should be some cleanup to clean majority of CET::Get() tbh, is fugly... So rather use that, so it is easier to find when the time comes. No need to change this now... Just dont introduce another getter for same thing please.